apps/web/src/api/hooks/notification-templates/useCreateTemplateFromBlueprint.ts

```diff
@@ -14,6 +14,7 @@ const mapBlueprintToTemplate = (blueprint: INotificationTemplate): ICreateNotifi
   critical: blueprint.critical,
   preferenceSettings: blueprint.preferenceSettings,
   blueprintId: blueprint._id,
+  __source: blueprint.__source,
```

Comment:
The `__source` is not a part of the `blueprint/template` it's something additional - metadata, then I feel like we shouldn't send it in the body, but in the query params instead.

Reply:
Done :)